posts/2020-08-12-open-liberty-tools-intellij.adoc

```diff
@@ -0,0 +1,87 @@
+blog_description: "The new Open Liberty Tools IntelliJ extension allows you to easily start Open Liberty development mode from within IntelliJ IDEA. You can make changes to your application with the server running, run tests and view test reports, and debug all from within IntelliJ."
+---
+= Announcing Open Liberty Tools for IntelliJ
+Kathryn Kodama <https://github.com/kathrynkodama>
+
+The new Open Liberty Tools IntelliJ extension provides you with an easier way to view and interact with your Open Liberty projects in IntelliJ IDEA.  
+
+You can start Open Liberty in dev mode, make changes to your application while the server is running, debug your application, and run tests and view the corresponding test reports all through the Liberty Dev Dashboard.
+
+In prior blogs, we have shown you how Open Liberty dev mode can be https://openliberty.io/blog/2019/10/22/liberty-dev-mode.html[run from a command line with Maven] and with https://openliberty.io/blog/2020/03/11/gradle-dev-mode-open-liberty.html[Gradle].  In this blog post, you will see how the Open Liberty Tools extension improves the developer experience by conveniently displaying Open Liberty projects and dev mode interactions within IntelliJ.  
+
+Check out a demo of Open Liberty Toosl for IntelliJ:
+++++
+<iframe width="560" height="315" src="https://www.youtube.com/embed/4qjHNYE7pSo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+++++
+
+== Installing the extension
+
+Install the https://plugins.jetbrains.com/plugin/14856-open-liberty-tools[Open Liberty Tools extension from the IntelliJ Plugins Marketplace].
+
+== Setting up the project
+
+For Maven projects, dev mode requires the Liberty Maven Plugin. Specify the latest version of the Liberty Maven Plugin in your project’s `pom.xml` file.
+[source,xml]
+----
+    <plugin>
+        <groupId>io.openliberty.tools</groupId>
+        <artifactId>liberty-maven-plugin</artifactId>
+        <version>3.2.2</version>
+    </plugin>
+----
+
+For Gradle projects, dev mode requires the Liberty Gradle Plugin. Specify the latest version of the Liberty Gradle Plugin in your project’s `build.gradle` file.
+[source,groovy]
+----
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.0'
+    }
+}
+----
+
+== Interacting with dev mode
+
+Once the Liberty Maven Plugin is specified in your `pom.xml` file, or the Liberty Gradle Plugin is specified in your `build.gradle` file, your project name will be listed under the Liberty Dev Dashboard.  You can interact with dev mode by right-clicking on your project name, or selecting a command from the drop-down below the project name.  Alternatively, you can use the "Run Liberty Dev Task” option in the dashboard toolbar.
```

Comment:
Don't need a comma in: Once the Liberty Maven Plugin is specified in your `pom.xml` file or the Liberty Gradle Plugin is specified in your `build.gradle` file

Reply:
Suggestion for second sentence: `You can interact with dev mode by right-clicking on your project name or by selecting a command from the drop-down [missing a noun here... e.g. "menu" or "list of commands"] below the project name.`